test/foundry/dapp/Application.t.sol

```diff
@@ -359,6 +373,13 @@ contract ApplicationTest is ERC165Test {
         bytes memory output = _getOutput(name);
         OutputValidityProof memory proof = _getProof(name);
 
+        _testERC721(output, proof);
+    }
+
+    function _testERC721(
```

Comment:
I understand you're aiming at minimal diffs for this PR for easier reviewing.
However, before merging, I think we should move the internal functions to the end of the contract.

Reply:
Yes. We should move internal functions to the end, unless it's easier to read at its current place. I think it might be easier to read here, but I don't mind moving it either

Reply:
It's just to keep things consistent, though I agree the function would be more distant to where it's being currently used.